charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -72,9 +73,9 @@ spec:
             - --cert=/var/etcd/ssl/client/tls.crt
             - --key=/var/etcd/ssl/client/tls.key
             - --cacert=/var/etcd/ssl/ca/ca.crt
-            - --endpoints=https://{{ .Values.name }}-local:{{ .Values.etcd.clientPort }}
+            - --endpoints=https://0.0.0.0:{{ .Values.etcd.clientPort }}
```

Comment:
This way, the livenessProbe is querying itself. Do you have any other suggestion here?

Reply:
Fine for me 👍

Reply:
What I forgot to mention: The server certificate for etcd needs to be replaced with this change since at least for Gardener `0.0.0.0` was not part of the `SAN` list. You should at least mention that explicitly in the release notes. Even better: Just change the endpoint in case of a multi-node etcd cluster to not break exiting single node clusters at all.